apps/web/src/app/(private)/api/v1/projects/[projectId]/commits/[commitUuid]/[...documentPath]/route.ts

```diff
@@ -0,0 +1,29 @@
+import { getDocumentByPath } from '@latitude-data/core'
+import apiRoute from '$/helpers/api/route'
+import { NextRequest } from 'next/server'
+
+export async function GET(
+  _: NextRequest,
+  {
+    params,
+  }: {
+    params: {
+      commitUuid: string
+      projectId: number
+      documentPath: string[]
+    }
+  },
+) {
+  return apiRoute(async () => {
```

Comment:
since nextjs does not allow post-route middlewares i've added this helper in order to easily handle errors in api endpoints. Not ideal but don't know if it can be done better with nextjs.